packages/app/src/client/views/login/index.tsx

```diff
@@ -175,17 +259,44 @@ const Login = (props: { state?: State }) => {
   );
 };
 
-const LoginPage = () => {
+function LoginPageParent() {
+  // Uninitialized state will cause Child to error out
+  const [loginConfig, setLoginConfig] = useState<LoginConfigInterface | undefined>();
+  useEffect(() => {
+    (async () => {
+      // Without this early exit criterion, `fetchLoginConfig()` is going to
+      // be called with every render cycle.
+      if (loginConfig !== undefined) {
+        return
+      }
+
+      const lcfg = await fetchLoginConfig();
+      setLoginConfig(lcfg);
+    })();
+  });
+
+  // Do not render child until loginconfig is populated. Kudos to
+  // https://stackoverflow.com/a/57312722/145400 for the `{...loginConfig}` to
+  // work around `not assignable to type 'IntrinsicAttributes..` kind of errors
+  // when doing `loginconfig={loginconfig}`.
+  return (
+    <div>{loginConfig && <LoginPageChild {...loginConfig} />}</div>
+  );
+}
+
+function LoginPageChild(lcfg: LoginConfigInterface) {
+  console.log('initialize LoginPageChild() with login config: ', lcfg)
   const [state, setState] = useState<State | undefined>();
 
   const onRedirectCallback = useCallback((state: AppState) => {
     setState(state as State);
   }, []);
+
   return (
     <Page>
       <Auth0Provider
-        domain="opstrace-dev.us.auth0.com"
-        clientId="vs6bgTunbVK4dvdLRj02DptWjOmAVWVM"
+        domain={lcfg.auth0_domain}
+        clientId={lcfg.auth0_client_id}
```

Comment:
Almost everything in this patch was so that this here can be done, in a robust fashion.